### PR TITLE
String Formatting fixed UI closes #2343

### LINKED ISSUE #2343: Fix string formatting issue for Test : tests.foreman.ui.test_gpgkey.GPGKey.test_consume_content_1
Failing due to:
Unicode formatting issue for string in AssertEqual function of Test.

StackTrace:
```
File "/usr/lib64/python2.7/unittest/case.py", line 369, in run
    testMethod()
  File "/home/jenkins/shiningpanda/jobs/abb3bf3f/virtualenvs/d41d8cd9/lib/python2.7/site-packages/ddt.py", line 146, in wrapper
    return func(self, *args, **kwargs)
  File "/home/jenkins/workspace/satellite6-automation-downstream-rhel66-ui/tests/foreman/ui/test_gpgkey.py", line 456, in test_consume_content_1
    .format(result.stderr, result.return_code)
'\'ascii\' codec can\'t encode character u\'\\u201c\' in position 298: ordinal not in range(128)
```

### PR DESCRIPTION
**Pull request by jyejare**
Fixed string format.

### CODE REVIEW

**Comment by jyejare** [comment]
Local passing Logs:
```
[root@jyejare robottelo]# nosetests -m test_consume_content_1 ./tests/foreman/ui/test_gpgkey.py
.
----------------------------------------------------------------------
Ran 1 test in 302.084s

OK
```

File: tests/foreman/ui/test_gpgkey.py [hunk 1]
```diff
@@ -452,7 +452,7 @@ def test_consume_content_1(self, key_name):
             )
             self.assertEqual(
                 result.return_code, 0,
-                'failed to fetch katello-ca rpm: {0}, return code: {1}'
+                u'failed to fetch katello-ca rpm: {0}, return code: {1}'
```
Comment (elyezer) [inline tests/foreman/ui/test_gpgkey.py]: :+1:

**Comment by elyezer** [comment]
ACK

**Comment by jyejare** [comment]
Closing this and adding the changes in 2345.

**Pull request closed without merging**